src/helpers/Sentry.ts

```diff
@@ -27,6 +27,11 @@ export default {
   initializeWithMiddleware(): Middleware {
     return (store) => {
       Sentry.init({
+        allowUrls: [
+          /localhost/,
+          /openstax.org/,
+          /https?:\/\/rex-web(.*\.)?\.herokuapp\.com/,
+        ],
```

Comment:
@TomWoodward I'm still confused about what this list is doing exactly. Some errors are definitely still reported for URLs not on the list (ie when I remove `localhost`), but it seems like errors that are manually reported via `Sentry.captureException` are filtered out.

Would it be worth first trying some custom event filters [here](https://sentry.io/settings/openstax/projects/rex/fi) before we rely on a whitelist or does that seem too limiting/unpredictable?

Reply:
[It seems like `allowURL` works only for captured events](whitelistUrls)
I think that maybe we want to use [`whitelistUrls`](https://docs.sentry.io/clients/javascript/tips/#decluttering-sentry)?

Reply:
[docs](https://docs.sentry.io/platforms/javascript/guides/angular/configuration/filtering/#decluttering-sentry):
_Prior to version 5.17.0, allowUrls and denyUrls were called whitelistUrls and blacklistUrls respectively. These options are still supported for backwards compatibility reasons, but they will be removed in version 6.0._ 

What is the reason behind switching to `allowUrls`?

Reply:
Okay, i see:)